genienlp/models/common.py

```diff
@@ -40,11 +40,6 @@
 from torch.nn.utils.rnn import pad_packed_sequence as unpack
 
 
-class EmbeddingOutput(NamedTuple):
```

Comment:
`TransformerDecoder` and `mask_tokens()` in `common.py` seem to be unused as well.